experimenter/experimenter/targeting/constants.py

```diff
@@ -2834,7 +2836,17 @@ def __post_init__(self):
         &&
         !{TOU_NOTIFICATION_BYPASS_ENABLED}
         &&
-        !{ADS_DISABLED}
+        (
+            (
+                (region == "MX" || region == "BR")
+                && !{ADS_DISABLED_BR_MX}
+            )
+            ||
+            (
+                (region != "MX" && region != "BR")
```

Comment:
This isn't really necessary makes maintenance / changes easier to inadvertently miss.  I'm concerned that as we add more and more constants, this line grows unnecessarily huge.

WDYT?